manifests/server/install.pp

```diff
@@ -1,5 +1,7 @@
 # == Class dns::server
 #
+include dns::server::default
```

Comment:
@Cicco0 yea I think that would work, but I think I actually prefer it to be in the config class, as it is a config file?